thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/ThreadComparisonPane.kt

```diff
@@ -85,7 +91,28 @@ class ThreadComparisonPane(
         threads.firstOrNull { it != null }?.let {
             header.setText(it)
         }
+
+        val onlyOneThread = threads.filterNotNull().size == 1
+
+        val highestCpu = if (!onlyOneThread) {
```

Comment:
Since you're always _inverting_ `onlyOneThread`, it reads cleaner if it's `val moreThanOneThread`; in my/IntelliJ's opinion :)